uni/pubspec.yaml

```diff
@@ -50,6 +51,7 @@ dependencies:
     git:
       url: https://github.com/joutvhu/open_file_plus.git
       ref: "3c32191"
+  openid_client: ^0.4.8
```

Comment:
Last update to this package [was 8 months ago](https://github.com/appsup-dart/openid_client), I'm a _tiny_ bit worried about its longetivity.

Reply:
Agree, I think we can remove that in the future and implement by hand the flow. It helps now to push something working to the beta and start testing

Reply:
We should not make the flow by hand. OpenID Connect authentication flow is a bit complex and might be non-trivial to implement. We can search for other packages though.